functions/src/send-email-to-manager.ts

```diff
@@ -68,13 +68,18 @@ export const sendEmailToManager = functions.firestore
      */
     let emailSent = false;
     await firebase.firestore().runTransaction(async transaction => {
+      //Make sure we didn't send a mail already
+      if (vote.emailToManagerSent) {
+        console.info("Email already sent to manager, aborting");
+        return;
+      }
       if (!emailSent) {
```

Comment:
Cette condition ne sert plus à rien vu qu'on verifie juste avant si il a été envoyé précédement non ? On peut du coup se passer de cette variable ?

Reply:
`emailToManagerSent` can tell if the email was sent by another run of the function. This one can tell if this run of the function has already sent it (the function passed to `runTransaction` can be called multiple times.